static/css/section/_cloud.scss

```diff
@@ -496,6 +490,51 @@
     }
   }
 
+  // @subsection /cloud/public-cloud
+  &.cloud-public-cloud {
```

Comment:
that's a body class that is constructed

![image](https://cloud.githubusercontent.com/assets/441217/20175907/241047bc-a73d-11e6-9dfe-2fba1dafe116.png)